pkg/controllers/deprovisioning/emptiness.go

```diff
@@ -55,7 +55,10 @@ func (e *Emptiness) ShouldDeprovision(ctx context.Context, n *state.Node, provis
 
 	emptinessTime, err := time.Parse(time.RFC3339, emptinessTimestamp)
```

Comment:
Musing: since the node is always passed into this interface method, can we always pass the node name into the logger encapsulated by the caller in the `controller.go` file?

Reply:
Sounds good! Updated the value to be in the `controller.go` file